<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-10T20:26:24 IST" name="Default suite" finished-at="2024-08-10T20:26:37 IST" duration-ms="13612">
    <groups>
    </groups>
    <test started-at="2024-08-10T20:26:24 IST" name="Default test" finished-at="2024-08-10T20:26:37 IST" duration-ms="13612">
      <class name="TestCases.LoginTest">
        <test-method is-config="true" signature="SetrupDriver()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:24 IST" name="SetrupDriver" finished-at="2024-08-10T20:26:29 IST" duration-ms="4809" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetrupDriver -->
        <test-method signature="LockedUser()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:29 IST" name="LockedUser" finished-at="2024-08-10T20:26:29 IST" duration-ms="786" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LockedUser -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:29 IST" name="TearDown" finished-at="2024-08-10T20:26:30 IST" duration-ms="173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="SetrupDriver()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:30 IST" name="SetrupDriver" finished-at="2024-08-10T20:26:32 IST" duration-ms="2351" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetrupDriver -->
        <test-method signature="LoginFailureTest()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:32 IST" name="LoginFailureTest" finished-at="2024-08-10T20:26:32 IST" duration-ms="502" status="FAIL">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<......> but was:<...Username...>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<......> but was:<...Username...>
at junit.framework.Assert.assertEquals(Assert.java:81)
at junit.framework.Assert.assertEquals(Assert.java:87)
at Pages.LoginPage.LoginFaliure(LoginPage.java:42)
at TestCases.LoginTest.LoginFailureTest(LoginTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginFailureTest -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:33 IST" name="TearDown" finished-at="2024-08-10T20:26:33 IST" duration-ms="155" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="SetrupDriver()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:33 IST" name="SetrupDriver" finished-at="2024-08-10T20:26:35 IST" duration-ms="2221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetrupDriver -->
        <test-method signature="loginSucessTest()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:35 IST" name="loginSucessTest" finished-at="2024-08-10T20:26:37 IST" duration-ms="2184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginSucessTest -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:TestCases.LoginTest@654f0d9c]" started-at="2024-08-10T20:26:37 IST" name="TearDown" finished-at="2024-08-10T20:26:37 IST" duration-ms="112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- TestCases.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
